From f378ebf24128f68985a6ce08f5f11c76d9b43618 Mon Sep 17 00:00:00 2001
From: Liao Junxuan <mikeljx@126.com>
Date: Tue, 11 Jul 2023 10:08:13 +0800
Subject: [PATCH 2/2] Fix Doxygen 1.9.7 build

Remove the deprecated tag DOT_TRANSPARENT because it causes an error on Doxygen
1.9.7.
---
 api/docs/API.doxy | 1 -
 1 file changed, 1 deletion(-)

diff --git a/api/docs/API.doxy b/api/docs/API.doxy
index eb06d7c..eb8ccf0 100644
--- a/api/docs/API.doxy
+++ b/api/docs/API.doxy
@@ -239,7 +239,6 @@ DOT_IMAGE_FORMAT       = png
 DOT_PATH               =
 DOTFILE_DIRS           =
 MAX_DOT_GRAPH_DEPTH    = 1000
-DOT_TRANSPARENT        = NO
 DOT_MULTI_TARGETS      = NO
 GENERATE_LEGEND        = YES
 DOT_CLEANUP            = YES
-- 
2.41.0

